proto/regen/data/v1alpha2/types.proto

```diff
@@ -0,0 +1,110 @@
+    //
+    // ## **General Rules**
+    //
+    // - only the WGS 84 reference coordinate system is supported and all coordinates should be assumed to be in WGS 84
+    // - only 2D and 3D coordinates are supported, coordinates with linear referencing systems or an “M” component are unsupported
+    //
+    // ## **`POLYGON` Rules**
+    //
+    // - exterior linear rings must be defined in counterclockwise direction
+    // - interior linear rings must be defined in clockwise direction
+    // - where there is more than one ring, the first MUST be the exterior ring and other rings MUST be linear rings - this means that the area of a polygon MUST be continuous and unbroken. To represent multiple non-contiguous shapes, a `MULTIPOLYGON` should be used instead. Ex: in the case where inside of a hole in a polygon, there is an island that we want to include. Because the island is disjoint, this shape must be represented as a `MULTIPOLYGON` instead of a `POLYGON` with a hole and an island in the hole
+    // - the first and last point in a linear ring must be identical
+    // - *canonical coordinate ordering rule*: the first (and last) point in a linear ring must be the point with the minimum X value in the ring. In the event where there is more than one point in a ring with the minimum X value, the first point must be the one with the minimum Y value. For 3D geometries, if there is is more than one point with the minimum X and Y values, the one with the minimum Z value should be the first.
+    // - if there is more than one interior linear ring, those rings should be ordered using the above *canonical coordinate ordering rule*, i.e. the ring with the minimum X coordinate comes first, etc.
+    //
+    // ## `**MULTIPOLYGON` Rules**
+    //
+    // - each polygon should conform to the above `POLYGON` rules
+    // - if there is more than one polygon, those polygons should be ordered using the above *canonical coordinate ordering rule*, i.e. the polygon with the minimum X coordinate comes first, etc.
+    ID_TYPE_GEOGRAPHY = 2;
+}
+
+
+// MediaType defines MIME media types to be used with ID_TYPE_RAW_UNSPECIFIED.
+enum MediaType {
+    MEDIA_TYPE_UNSPECIFIED = 0;
+
+    // basic formats
+    MEDIA_TYPE_TEXT_PLAIN = 1;
+    MEDIA_TYPE_JSON = 2;
+    MEDIA_TYPE_CSV = 3;
+    MEDIA_TYPE_XML = 4;
+    MEDIA_TYPE_PROTOBUF_ANY = 5;
+    MEDIA_TYPE_PDF = 6;
+
+    // images
+    MEDIA_TYPE_TIFF = 16;
+    MEDIA_TYPE_JPG = 17;
+    MEDIA_TYPE_PNG = 18;
+    MEDIA_TYPE_SVG = 19;
+    MEDIA_TYPE_WEBP = 20;
+    MEDIA_TYPE_AVIF = 21;
+    MEDIA_TYPE_GIF = 22;
+    MEDIA_TYPE_APNG = 23;
+
+    // audio-visual media containers
+    MEDIA_TYPE_MPEG = 32;
+    MEDIA_TYPE_MP4 = 33;
+    MEDIA_TYPE_WEBM = 34;
+    MEDIA_TYPE_OGG = 35;
+
+    // geospatial raster formats
+    // TODO
+
+    // geospatial vector formats
+    // TODO
```

Comment:
This means that we could support both geo data as `ID_TYPE_RAW_UNSPECIFIED` but also as `ID_TYPE_GEOGRAPHY`. Could you think of use cases where we'd use one over another? Could that be actually relevant to use both in some cases? That also relates a bit to point 3. of @clevinson's comment: https://github.com/regen-network/regen-ledger/pull/221#pullrequestreview-572931219.

Reply:
I am thinking of `ID_TYPE_GEOGRAPHY` to represent a concrete polygon representing a project area.

But, we will likely also want annotated map data using a format like https://docs.mapbox.com/vector-tiles/reference/. I think we'd just want to store the raw hashed tile data rather than worrying about canonicalization.

I'm more thinking of `ID_TYPE_GEOGRAPHY` objects for use in signed claims, whereas map tile data, photos, videos, etc.  are more informational and less involved in verifying claims.

Whta do you think?

Reply:
Yeah it makes sense. There might be cases where some data could be used both for verifying claims and for informational purposes, e.g. a project boundaries. In this case, should we just use `ID_TYPE_GEOGRAPHY`?